cli/tsc/dts/lib.deno.unstable.d.ts

```diff
@@ -764,6 +765,31 @@ declare namespace Deno {
     symbols: Const<S>,
   ): DynamicLibrary<S>;
 
+  /** **UNSTABLE**: New API, yet to be vetted.
+   *
+   *  Creates a presentable WebGPU surface from given window and
+   *  display handles.
+   *
+   *  The parameters correspond to the table below:
+   *
+   *  | system            | winHandle     | displayHandle   |
+   *  | ----------------- | ------------- | --------------- |
+   *  | "cocoa" (macOS)   | `NSView*`     | -               |
+   *  | "win32" (Windows) | `HWND`        | `HINSTANCE`     |
+   *  | "x11" (Linux)     | Xlib `Window` | Xlib `Display*` |
+   *
+   * @category WebGPU
```

Comment:
I wonder if this is really the right category for this. lets land as-is for now, but lets revisit once we have canvas